apps/question-bank/src/main/resources/application.yml

```diff
@@ -1,8 +1,13 @@
 nbs:
   datasource:
     server: localhost
-
-
+  security:
+    tokenSecret: qJ5+oDurrJnxfaAz6h4Eg1saP7NSeHuQQ/Yq4vh+mbi4B4QYThc/M+33VtPJ5O2A
+    tokenIssuer: gov.cdc.nbs
+    tokenExpirationMillis: 1800000 # 30 minutes
+    parameterSecret: Yq4vh+mbi4B4QYThc/M+33VtPJ5O2Adc
```

Comment:
i think `parameterSecret` is only used for the encryption endpoint, prob don't need it.